client/README.md

```diff
@@ -0,0 +1,42 @@
+# Wagtail client-side components
+
+This library aims to give developers the ability to subclass and configure Wagtail's UI components.
+
+## Usage
+
+```
+npm install wagtail
+```
+
+```javascript
+import { Explorer } from 'wagtail';
+
+...
+
+<Explorer onChoosePage={(page)=> { console.log(`You picked ${page}`); }} />
+
+```
+
```

Comment:
Maybe a Features heading or similar where we list things like eslint, React, ES6, fetch etc to give devs a quick primer on what to expect and what editor plugins would help?